<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YLPAINT</class>
 <widget class="QMainWindow" name="YLPAINT">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>YL.PAINT</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_save"/>
    <addaction name="action_open"/>
    <addaction name="action_files_history"/>
    <addaction name="separator"/>
    <addaction name="action_clear"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="action_full_screen"/>
    <addaction name="action_show_as_window"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="tool_bar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="action_clear">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="action_circle">
   <property name="text">
    <string>Круг</string>
   </property>
  </action>
  <action name="action_square">
   <property name="text">
    <string>Квадрат</string>
   </property>
  </action>
  <action name="action_triangle">
   <property name="text">
    <string>Треугольник</string>
   </property>
  </action>
  <action name="action_full_screen">
   <property name="text">
    <string>Полный экран</string>
   </property>
  </action>
  <action name="action_show_as_window">
   <property name="text">
    <string>В окне</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="action_files_history">
   <property name="text">
    <string>История</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
